dataplane/luciusctl/luciusctl.go

```diff
@@ -12,21 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package luciusctl
 
 import (
-	"os"
-
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
 	"github.com/openconfig/lemming/dataplane/luciusctl/info"
 	"github.com/openconfig/lemming/dataplane/luciusctl/sai"
 )
 
-func main() {
-	cmd := cobra.Command{
-		Use: "luciusctl",
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lucius",
```

Comment:
should this still be luciusctl?

Reply:
i thought that was too many characters to type